articles/hooks/extensibility-points/send-phone-message.md

```diff
@@ -117,8 +117,9 @@ The `context.language` parameter will always have one of the languages configure
 
 # Keep reading
 
-- [Configuring a Custom SMS Provider for MFA using Amazon SNS](/multifactor-authentication/send-phone-message-hook-amazon-sns) guide. 
+- [Configuring a Custom SMS Provider for MFA using Amazon SNS](/multifactor-authentication/send-phone-message-hook-amazon-sns) guide.
```

Comment:
The contents for customizing SMS providers via hooks are really difficult to discover from MFA docs. Is it possible to add additional links to these provider on pages like: https://auth0.com/docs/mfa/guides/configure-sms, such that customers looking at MFA will be able to find out the other options more easily?

Reply:
Good call. I'll add some interlinking there.